src/WindowsUtils/BusyLoopLauncher.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "WindowsUtils/BusyLoopLauncher.h"
+
+namespace orbit_windows_utils {
+
+BusyLoopLauncher::BusyLoopLauncher() : debugger_({this}) {}
+
+ErrorMessageOr<BusyLoopInfo> BusyLoopLauncher::StartWithBusyLoopAtEntryPoint(
```

Comment:
From an API point of view. What happens when `StartWithBusyLoopAtEntryPoint` gets called twice? As far I can see this will mess up with the `busy_loop_or_error_promise_`, right? I think this should be fixed, at least by added some checks+documentation to explicitly forbid this usecase.

Reply:
Yeah it was meant to be single-use. I added a "state_" variable that makes sure the functions are called only once and in the right order.

Reply:
Thanks.